assets/files.wxs

```diff
@@ -1958,6 +1958,14 @@
             <File Id="fil6B026F6D5210498B99E5F14386F4F90B" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\Microsoft.PowerShell.ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-windows`
Validate Remove-LocalGroupMember cmdlet.Can remove array of user SIDs from group
```powershell
queryFilter
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 527
527:             Remove-LocalGroupMember TestGroupRemove1 -Member @($user1sid, $user2sid)

```
Validate Remove-LocalGroupMember cmdlet.Can remove array of users names or SIDs from group
```powershell
queryFilter
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 534
534:             Remove-LocalGroupMember TestGroupRemove1 -Member @($user1sid, "TestUserRemove2")

```
Validate Add-LocalGroupMember cmdlet.Can add array of user SIDs to group
```powershell
queryFilter
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 173
173:             Add-LocalGroupMember TestGroup1 -Member @($user1sid, $user2sid)

```
Validate Add-LocalGroupMember cmdlet.Can add array of users names or SIDs to group
```powershell
queryFilter
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 181
181:             Add-LocalGroupMember TestGroup1 -Member @($user1sid, "TestUser2")

```